### ISSUE TITLE
Use `explicit` for single argument constructors

### ISSUE THREAD

**Issue by bylaws** [skyline-emu/skyline#171]
This would to prevent unintended logic errors by preventing accidental construction.

**Comment by PixelyIon** [comment]
No, it will be done at some point still. We haven't gotten a proper point to refactor due to `ftx1` having so much in flux so it'll probably be done when we're merging it.

I should add that all this affects is code quality and not anything that might be visible to the end user so if you're concerned about it in that way then this is irrelevant.
